README.md

```diff
@@ -1 +1,161 @@
-# node-modules-path
\ No newline at end of file
+# Paths, Folders and Modules
+
+## Overview
+
+This lesson will cover how to work with paths, use them when importing modules. We'll also show a code organization technique where you can import folders.
+
+## Objectives
+
+1. Describe `__dirname`, `process.cwd()` and `process.env.PWD`
+1. Describe how to import folders (and why)
+
+
+## Absolute Path with __dirname
+
+Consider the code in which we read a file on Mac OS X. The file is in a `data` folder relative to our script:
+
+```js
+var fs = require('fs')
+var customers = fs.readFileSync('data/customers.csv', {encoding: 'utf8'})
+console.log(customers)
+```
+
+Unfortunately, this code will break on Windows. This is due to a different path syntax, i.e., `data\customers.csv`. That's why we use `path.join()` which we discussed in one of the previous lessons. This example is in the file `customers.js`:
```

Comment:
"that's why we use `path.join()` so that we can control the path to files and directories". we like each piece of curriculum to stand alone and not necessarily reference previous lessons (makes it harder to reuse curriculum in other circumstances)